ballerina/tests/mysql_test_entities_generated_client.bal

```diff
@@ -152,30 +152,30 @@ public isolated client class SQLTestEntitiesClient {
     };
 
     public isolated function init() returns persist:Error? {
-        mysql:Client|error dbClient = new (host = host, user = user, password = password, database = database, port = port);
+        mysql:Client|error dbClient = new (host = mysql.host, user = mysql.user, password = mysql.password, database = mysql.database, port = mysql.port);
         if dbClient is error {
             return <persist:Error>error(dbClient.message());
         }
         self.dbClient = dbClient;
         self.persistClients = {
-            [ALL_TYPES] : check new (self.dbClient, self.metadata.get(ALL_TYPES)),
-            [STRING_ID_RECORD] : check new (self.dbClient, self.metadata.get(STRING_ID_RECORD)),
-            [INT_ID_RECORD] : check new (self.dbClient, self.metadata.get(INT_ID_RECORD)),
-            [FLOAT_ID_RECORD] : check new (self.dbClient, self.metadata.get(FLOAT_ID_RECORD)),
-            [DECIMAL_ID_RECORD] : check new (self.dbClient, self.metadata.get(DECIMAL_ID_RECORD)),
-            [BOOLEAN_ID_RECORD] : check new (self.dbClient, self.metadata.get(BOOLEAN_ID_RECORD)),
-            [COMPOSITE_ASSOCIATION_RECORD] : check new (self.dbClient, self.metadata.get(COMPOSITE_ASSOCIATION_RECORD)),
-            [ALL_TYPES_ID_RECORD] : check new (self.dbClient, self.metadata.get(ALL_TYPES_ID_RECORD))
+            [ALL_TYPES] : check new (self.dbClient, self.metadata.get(ALL_TYPES), MYSQL_SPECIFICS),
+            [STRING_ID_RECORD] : check new (self.dbClient, self.metadata.get(STRING_ID_RECORD), MYSQL_SPECIFICS),
+            [INT_ID_RECORD] : check new (self.dbClient, self.metadata.get(INT_ID_RECORD), MYSQL_SPECIFICS),
+            [FLOAT_ID_RECORD] : check new (self.dbClient, self.metadata.get(FLOAT_ID_RECORD), MYSQL_SPECIFICS),
+            [DECIMAL_ID_RECORD] : check new (self.dbClient, self.metadata.get(DECIMAL_ID_RECORD), MYSQL_SPECIFICS),
+            [BOOLEAN_ID_RECORD] : check new (self.dbClient, self.metadata.get(BOOLEAN_ID_RECORD), MYSQL_SPECIFICS),
+            [COMPOSITE_ASSOCIATION_RECORD] : check new (self.dbClient, self.metadata.get(COMPOSITE_ASSOCIATION_RECORD), MYSQL_SPECIFICS),
+            [ALL_TYPES_ID_RECORD] : check new (self.dbClient, self.metadata.get(ALL_TYPES_ID_RECORD), MYSQL_SPECIFICS)
         };
     }
 
     isolated resource function get alltypes(AllTypesTargetType targetType = <>) returns stream<targetType, persist:Error?> = @java:Method {
-        'class: "io.ballerina.stdlib.persist.sql.datastore.MySQLProcessor",
+        'class: "io.ballerina.stdlib.persist.sql.datastore.SQLProcessor",
```

Comment:
this could be a breaking change as generated code in update 06 will not work right?

Reply:
Fixed in [d653e9b](https://github.com/ballerina-platform/module-ballerinax-persist.sql/pull/14/commits/d653e9b9c20dc15153cc0dd37784e675e850b6e1)